wafer/talks/views.py

```diff
@@ -51,6 +52,7 @@ def get_object(self, *args, **kwargs):
         '''Only talk owners can see talks, unless they've been accepted'''
         object_ = super(TalkView, self).get_object(*args, **kwargs)
         if object_.can_view(self.request.user):
+            object_.private_notes = mark_safe(object_.private_notes)
```

Comment:
Between Django's "safe" concept, the fact that private notes are limited to organisers and the benefit of HTML in the notes (not at all required), I considered the attack vectors to be negligible. But we certainly don't have to allow HTML parsing.